### PR TITLE
chore: release golang cli on linux

### PR DESCRIPTION
**Pull request by Avishagp**
Release golang CLI on linux.
[JIRA ticket](https://snyksec.atlassian.net/browse/HMMR-567)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1054,6 +1088,10 @@ workflows:
             - Build (snyk-linux-arm64)
           go_os: linux
           go_arch: arm64
+      - v2-rename-linux-artifact:
+          name: v2 / Rename linux/amd64 artifact
+          requires:
+            - v2 / Sign (linux/amd64)
```
Comment (PeterSchafer) [inline .circleci/config.yml:1094]: Question: I think this should be "Build (linux/amd64)"

File: .circleci/config.yml [hunk 2]
```diff
@@ -1140,8 +1178,8 @@ workflows:
           name: v2 / Regression Tests (linux/amd64)
           context: nodejs-install
           requires:
-            - v2 / Build (linux/amd64)
-          test_snyk_command: /mnt/ramdisk/snyk/cliv2/bin/snyk_linux_amd64
+            - v2 / Rename macos/amd64 artifact
```
Comment (PeterSchafer) [inline .circleci/config.yml:1181]: Question: why is this requiring "v2 / Rename macos/amd64 artifact"? Shouldn't it be linux?

**Review by PeterSchafer — CHANGES_REQUESTED** [review]
Unfortunately the build doesn't run, please see my comments on how to fix it.

File: .circleci/config.yml (hunk 2 above)
Comment (Avishagp) [reply .circleci/config.yml:1181]: My bad!

File: .circleci/config.yml (hunk 1 above)
Comment (Avishagp) [reply .circleci/config.yml:1094]: I used the same logic from `v2-rename-windows` and `v2-rename-macos`, both require signing before renaming. The logic is sign -> rename -> build. Does that make sense?

Comment (PeterSchafer) [reply .circleci/config.yml:1094]: In general that would be right, but linux artifacts don't get signed right now. 

If you look at the build log https://app.circleci.com/pipelines/github/snyk/cli/16176, you can see that this unknown job is the problem of your build.

You can also look at your changes here: https://github.com/snyk/cli/pull/4255/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L1087 the prerequisite for the tests was "build" not "sign". So this should also be the prerequisite for the "rename".

I hope this helps.

Comment (Avishagp) [reply .circleci/config.yml:1094]: I'm sorry! thanks for explaining that

**Comment by PeterSchafer** [comment]
Please also take rename/replace "snyk_linux_arm64"

**Review by PeterSchafer — APPROVED** [review]

**Pull request merged**